app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -330,6 +402,31 @@ class StoryFragmentTest {
     }
   }
 
+  private fun createStoryActivityIntent(): Intent {
```

Comment:
Prefer: ``createFractionsStoryActivityIntent`` to contrast from the test one below. Or possibly better: have a single 'createStoryActivityIntent' that takes the topic & story IDs to provide exact context at the callsite.

Reply:
There's no point having a `createStoryActivityIntent` as a separate function - all it does is call another function. I think it was created to remove code duplication, which we don't want to re-introduce. I like the new name - `createFractionsStoryActivityIntent `, I'll go with that option.

Reply:
resolving on behalf of Ben.